Source: Stefan Kost <ensonic@users.sf.net>
Upstream: Yes, included in v1.16
Reason: This bug causes many packages to fail when building gtk-doc documentation.

See https://bugzilla.gnome.org/show_bug.cgi?id=627223
---
From 2bffebcf9c125f5610b8fb660d42c3c5b9dfd4f0 Mon Sep 17 00:00:00 2001
From: Stefan Kost <ensonic@users.sf.net>
Date: Thu, 19 Aug 2010 17:45:54 +0300
Subject: [PATCH] fixxref: tweak the vim invocation

Use specific output-filename to ensure we get what we'll late use. Use "-u NONE"
instead of -u /dev/null.
Fixes #627223
---
 gtkdoc-fixxref.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/gtkdoc-fixxref.in b/gtkdoc-fixxref.in
index 33a31b8..9f53852 100755
--- a/gtkdoc-fixxref.in
+++ b/gtkdoc-fixxref.in
@@ -464,7 +464,7 @@ sub HighlightSourceVim {
     close (NEWFILE);
 
     # format source
-    system "echo 'let html_number_lines=0|let html_use_css=1|let use_xhtml=1|syn on|e $temp_source_file|run! syntax/2html.vim|wa!|qa!' | @HIGHLIGHT@ -n -e -u /dev/null -T xterm >/dev/null";
+    system "echo 'let html_number_lines=0|let html_use_css=1|let use_xhtml=1|syn on|e $temp_source_file|run! syntax/2html.vim|w! $temp_source_file.html|qa!' | @HIGHLIGHT@ -n -e -u NONE -T xterm >/dev/null";
 
     my $highlighted_source;
     {
-- 
1.7.3.1

